erpnext/accounts/doctype/journal_entry/journal_entry.json

```diff
@@ -1124,10 +1124,10 @@
    "in_global_search": 0, 
    "in_list_view": 0, 
    "in_standard_filter": 0, 
-   "label": "Letter Head", 
+   "label": "LetterHead", 
```

Comment:
You need to change "modified" date-stamp as well in all json, otherwise it will not be reloaded with `bench update`.